templates/_components/button.twig

```diff
@@ -21,8 +21,8 @@
 {# @var string|null href #}
 {% set href = href ?? null %}
 
-{# @var boolean newWindow #}
-{% set newWindow = newWindow ?? false %}
+{# @var boolean newTab #}
+{% set newTab = newTab ?? false %}
```

Comment:
Craft calls their field option "new tab" so I thought I'd go with the flow.

![CleanShot 2025-01-03 at 10 37 41@2x](https://github.com/user-attachments/assets/e480cdac-bed7-4aa3-8d25-373e8640ae00)